src/index.ts

```diff
@@ -344,6 +355,9 @@ class Globe {
     locations.forEach((location) => {
       const position = this.calculateXYZFromLatLon(location.lat, location.lon);
 
+      // Offset location position slightly, so the angle is not through the globe dots
+      position.y = position.y > 0 ? position.y + 2 : position.y - 2;
+
       const mesh = new THREE.Mesh(geometry, material);
       mesh.position.copy(position);
       mesh.lookAt(targetVector);
```

Comment:
```suggestion
      const mesh = new THREE.Mesh(geometry, material);

      // Offset position to prevent shader intersection with globe dots
      mesh.position.copy(position).multiplyScalar(1.0025);
      mesh.lookAt(targetVector);
```

Reply:
Yup better to offset. Maintains better positioning.